### ISSUE TITLE
copy/paste into other applications should preserve syntax coloring

### ISSUE THREAD

**Issue by SethTisue** [NetLogo/NetLogo#321]
currently if you copy from NetLogo and paste into e.g. Word you just get ordinary black text

**Comment by ERSUCC** [comment]
I strongly disagree with this. It's often an issue I have with copying from applications like Word, because I end up with mismatched font sizes and colors when I paste the copied text. Especially annoying if I'm just trying to copy a single line or a few tokens, as opposed to a whole section of code. This also would present an issue with light/dark mode, as well as the future planned custom color themes.

**Comment by SethTisue** [comment]
(Might there be some way to offer it as an option, even if it wasn't the default?)

**Comment by ERSUCC** [comment]
Yeah, I would support adding this to the Edit menu. Something like "Copy with Colors" or "Copy with Highlighting". Might be a difficult change but should certainly be possible.

**Comment by TheBizzle** [comment]
Nah, just one way of copying.  If you don't want formatting, that's handled at the paste site.  Usually, `Ctrl+v` does a formatted paste, and `Ctrl+Shift+v` does an unformatted paste.

**Comment by ERSUCC** [comment]
This functionality has been implemented since commit [4b28794](https://github.com/NetLogo/NetLogo/commit/4b28794b6285f86774b6f7c0ea51d3134d640065), which updated `RSyntaxTextArea` to version 3.3.0.
